packages/lib-classifier/src/store/WorkflowStepStore.spec.js

```diff
@@ -121,7 +133,8 @@ describe('Model > WorkflowStepStore', function () {
 
     it('should convert the tasks to steps and set the steps', function () {
       const { workflowSteps } = ROOT_STORE_INSTANCE
-      expect(ROOT_STORE_INSTANCE.workflows.active.steps).to.have.lengthOf(WORKFLOW.steps.length)
+      const stepsSize = workflowSteps.steps.size
```

Comment:
Having both `workflows.active.steps` and `workflowSteps.steps` in the models is confusing.

Reply:
Yes, I agree, but I'd like to try to keep this scoped only to changes needed to get MobX updates working.

Reply:
No problem. I've been trying to figure out why this line changed and I'm still not 100% sure what's going on in this test, or what it was testing before it changed.

Reply:
Yes it was a mistake, likely caused by the similar naming in the two stores as you pointed out. I don't recall if something else was failing with it with the MobX update, but I likely took a closer look at it because it was failing and then realized it was testing the wrong thing to begin with.